internal/pkg/service/stream/components.go

```diff
@@ -21,6 +23,13 @@ type Component string
 
 func StartComponents(ctx context.Context, d dependencies.ServiceScope, cfg config.Config, components map[Component]bool) error {
 	// Start components, always in the same order
+
+	if components[ComponentDiskWriter] {
+		if err := writernode.Start(ctx, d, cfg); err != nil {
+			return err
+		}
+	}
+
```

Comment:
To create a sink, we need to create at least one file, with at least one slice.

To create slice, we need at least one active disk writer volume.

So in API E2E test, we have to start minimal disk writer component, which registers the volume.